.husky/pre-commit

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+# Get the current branch name
+BRANCH_NAME=$(git rev-parse --abbrev-ref HEAD)
+
+# Calculate the length of the branch name
+BRANCH_NAME_LENGTH=${#BRANCH_NAME}
+
+# Check if the length is greater than 37
+if [ $BRANCH_NAME_LENGTH -gt 37 ]; then
```

Comment:
I think we can also update this check to handle the character type limitation as well with some regex-foo:

> must contain only lowercase letters, numbers and dashes

Would anyone like to take a stab at integrating that?

Reply:
Thanks @Andrew565! This is just a bash script so there should be a pretty standard way to integrate a regex. https://devhints.io/bash